src/Test/WordPress/HooksTest.php

```diff
@@ -42,9 +42,10 @@ public function setup()
         $this->mockDefaultIntegration = new DefaultIntegration($this->mockConfig, $this->mockWordPressAPI, $this->mockDataStore, $this->mockLogger);
 
         $this->hooks = $this->getMockBuilder('\CF\WordPress\Hooks')
-            ->setMethods(array('setHooks'))
+            ->disableOriginalConstructor()
+            ->setMethods(array('__construct')) // This is a hack to make the tests work
```

Comment:
I needed to avoid constructor being called but not mocking other functions. Mocking a random function in this case `'__construct'` solved it.